lib/token.ts

```diff
@@ -34,15 +35,37 @@ export const DISPLAY_TOKENS = (
 }
 
 /**
- * Resolve the token address for a given mint across all chains
- * @param mint - The mint address
- * @returns The token address
+ * Returns the default quote token for a given mint and exchange on the chain of the mint
+ */
+export function getDefaultQuote(mint: `0x${string}`, exchange: Exchange) {
+  const chain = resolveAddress(mint).chain
+  if (!chain) {
+    return Token.create("UNKNOWN", "UNKNOWN", mint, 18, {})
```

Comment:
nit: we can use the `DEFAULT_TOKEN` from the `Token` package for this, right?

Reply:
it's not exported. in the future when it's exported i'm happy to make this change